src/lambda.cpp

```diff
@@ -19,13 +19,15 @@
 // lambda.cpp: Main File for Lambda
 // ==========================================================================
 
+#include "seqan2seqan3.hpp"
+
 #include <seqan3/argument_parser/all.hpp>
 
-#if 0
 #include "search.hpp"
-#endif
-#include "mkindex.hpp"
 
+#if 1
+#include "mkindex.hpp"
```

Comment:
Does the search now always get included? And why did that change? I guess the macro was initially meant to distinguish between search and index

Reply:
It's a leftover from debugging!